library/item/item_content_registry/src/main/java/org/quiltmc/qsl/item/content/registry/api/ItemContentRegistries.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2022 QuiltMC
+ * Copyright 2023 QuiltMC
```

Comment:
Keep it 2022; 2022-2023 won't really be necessary since yeah, we're moving away from the gray-area ranges
```suggestion
 * Copyright 2022 QuiltMC
```